client/js/components/user/DpOrganisationList/DpOrganisationFormFields.vue

```diff
@@ -30,7 +30,7 @@
           :id="organisation.id + ':orgaName'"
           class="w-full u-mb-0_5 block"
           style="height: 27px;"
-          data-cy="orgaName"
+          data-cy="orgaFormField:orgaName"
```

Comment:
If You do something like that, please inform the testers about the changes, when merging this

Reply:
yes, I just added a comment in this PR and wanted to inform the testers

Reply:
Testing approved